src/Gallery/Model/Image.ts

```diff
@@ -1,20 +1,17 @@
-export type ImageData = {
-  _id: string;
-  fileId: string;
-  name: string;
-  file: FormData;
+import { GalleryItem } from './GalleryItem';
+
+export type ImageItemData = GalleryItem & {
   tags: string[];
   suggestedTags: string[];
-  date: Date;
   defaultAlt: string;
   defaultCaption: string;
```

Comment:
I think all of these can go on the gallery item and we shouldn't need the ImageItemData.
I guess that would mean the ImageData can move out of the gallery and that would make a lot of sense.